frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/templates/install-config.hbs

```diff
@@ -306,8 +306,10 @@ publish: Internal
   openstack:
     cloud: {{{../../cloud}}}
     externalNetwork: {{{../../externalNetworkName}}}
-    lbFloatingIP: {{{../../apiFloatingIP}}}
-    ingressFloatingIP: {{{../../ingressFloatingIP}}}
+    {{#if ../../lbFloatingIP}}
+    lbFloatingIP: {{{../../lbFloatingIP}}}
+    {{/if}}
+    apiFloatingIP: {{{../../apiFloatingIP}}}
```

Comment:
Do not remove `ingressFloatingIP`, and we should only include `lbFloatingIP` or `apiFloatingIP` depending on OCP version.